client/reducers/editReducer.js

```diff
@@ -88,7 +88,12 @@ export function editReducer (state = initialState, action) {
       const { section, sectionIndex } = action.payload
       const article = cloneDeep(state.article)
 
-      article.sections.splice(sectionIndex, 0, section)
+      if (article.sections) {
+        article.sections.splice(sectionIndex, 0, section)
+      } else {
+        article.sections = [section]
+      }
```

Comment:
Cool, does this fix https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=4&projectKey=GROWTH&modal=detail&selectedIssue=GROWTH-311&sprint=27 ?

Reply:
Yes!